packages/slice-machine/src/modules/availableCustomTypes/index.ts

```diff
@@ -182,18 +182,14 @@ export const availableCustomTypesReducer: Reducer<
       const id = action.payload.customTypeId;
       const newName = action.payload.newCustomTypeName;
       const newLocalCustomType = { ...state[id].local, label: newName };
-      const isCustomTypeDisconnected =
-        newLocalCustomType.__status === CustomTypeStatus.UnknownDisconnected;
```

Comment:
now that I think about it, we should do that if it's any Unknown.

Reply:
Not for the push, but for the Save.
If the user is not connected to Prismic it wuld also write "New" or maybe i'm wrong and we already handle that case.